unreal_qt_stylesheet/main.py

```diff
@@ -0,0 +1,61 @@
+# import sys
+import os
+
+
+try:
+    from PySide2 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+try:
+    from PyQt5 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+try:
+    from PySide6 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+try:
+    from PyQt6 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+# from Qt import QtWidgets, QtCore, QtGui
+# from Qt import _loadUi
+from pathlib import Path
+
+
+# MODULE_PATH = os.path.dirname(os.path.abspath(__file__))
+# UI_PATH = os.path.join(MODULE_PATH, 'ui', 'progress.ui')
+# QSS_PATH = os.path.join(MODULE_PATH, 'ue.qss')
+MODULE_PATH = Path(os.path.abspath(__file__)).parent
+QSS_PATH = MODULE_PATH / 'ue.qss'
+
+
+def setup():
+    """ Apply the Unreal dark stylesheet to the current QApplication """
+    QtWidgets.QApplication.setAttribute(QtCore.Qt.AA_EnableHighDpiScaling)
+    app = QtWidgets.QApplication.instance()
+    QtCore.QResource.registerResource(str(MODULE_PATH / "icons.rcc") )
+    with open(QSS_PATH, 'r') as f:
+        qss = f.read()
+        app.setStyleSheet(qss)
+
+
+# if __name__ == '__main__':
```

Comment:
you can just remove all the commented code that I originally wrote

Reply:
will do

Reply:
This code in there, was that just a sample window with a loading-bar to show of the stylesheet?
```python
window = QtWidgets.QMainWindow()
_loadUi(UI_PATH, window)
window.show()
```

Reply:
yup, if you think it's helpful you could add it to the module-level docstring